e2e/internal/kubeclient/portforward.go

```diff
@@ -20,6 +21,8 @@ import (
 func (k *Kubeclient) WithForwardedPort(ctx context.Context, namespace, podName, remotePort string, f func(addr string) error) error {
 	var funcErr error
 	for i := range 3 {
+		// Apply backoff after first attempt.
+		time.Sleep(time.Duration(i) * time.Second)
```

Comment:
The `ctx` may expire before the sleep completes. Do we care about this?

Reply:
Maybe not at the current backoff levels, but you're right that it should be handled - let me fix that.